### DISCUSSION TITLE
How can i get the entity for "status_text" of Inverter and Battery?

### DISCUSSION THREAD

**Discussion by Stony111** [WillCodeForCats/solaredge-modbus-multi#598] [Q&A]
Hi, i have read the documentation and found the chapter "Entity Status Cards from Attributes". Can anyone tell me, in which config file i have to store the described settings? Is this a section in configuration.yaml?
Overall, the AddOn works very well, but there is no entity for ‘status_text’. The entity for the ‘status’ contains the ENUMs for e.g. I_STATUS_MPPT and not the actual status value e.g. 4!
Therefore, the question would be how I get the entity ‘status_text’!
Many thanks in advance for any tips.
Best Regards
Martin

**Comment by Ashpork** [comment]
Hi Martin,

Have a look at your I1 status entity, it might have the attribute you you're looking for.

![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/70691313/ea6a9f61-e996-4a58-959d-8c228191c4a9)

The example configuration shown on the "Entity Status Cards from Attributes" page in the documentation, this would be entered into a dashboard card. However the examples won't show you the value 4, you'll need to write a template and insert that into the card configuration.

**Comment by Stony111** [comment]
Thank you for the quick response. I have also made some progress in the meantime.
The section mentioned in the documentation can be entered directly in RAW-Editor in the dashboard for a card configuration, i.e. there is actually no ‘status_text’ entity at all, it is only prepared accordingly for the display. I wasn't aware of this, but it works for me too.
